GliaWidgets/Component/Header/Header.swift

```diff
@@ -85,12 +77,13 @@ class Header: UIView {
         closeButton.isHidden = true
     }
 
-    private func setup() {
+    override func setup() {
+        super.setup()
         effect = .none
-        backgroundColor = style.backgroundColor
+        backgroundColor = .blue
 
         titleLabel.font = style.titleFont
-        titleLabel.textColor = style.titleColor
```

Comment:
Is it ok you removed applying colors from style?

Reply:
fixed [9fbdd03](https://github.com/salemove/ios-sdk-widgets/pull/370/commits/9fbdd03de40296523e49e199fa156c61ff331ee1)

Reply:
I am still seeing `backgroundColor = .blue` and `titleLabel.textColor = .white` instead of the previous style colors

Reply:
my fail, fixed - d2d38d9f48ffc881ab7990da4c6bf5794050bb1f